new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -283,24 +283,24 @@ public void testDomainOnPvUsingWlst() throws IOException {
     // check admin server pod is ready
     logger.info("Waiting for admin server pod {0} to be ready in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodReady(adminServerPodName);
+    checkPodReady(adminServerPodName, domainUid, domainNamespace);
 
     // check managed server pods are ready
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
           managedServerPodNamePrefix + i, domainNamespace);
-      checkPodReady(managedServerPodNamePrefix + i);
+      checkPodReady(managedServerPodNamePrefix + i, domainUid, domainNamespace);
     }
 
     logger.info("Check admin service {0} is created in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkServiceCreated(adminServerPodName);
+    checkServiceExists(adminServerPodName, domainNamespace);
```

Comment:
This has to be done right after checkPodReady(adminServerPodName, domainUid, domainNamespace);

Reply:
done